tests/measure/conftest.py

```diff
@@ -0,0 +1,87 @@
+"""Fixtures for testing the measure module."""
+
+import numpy as np
+import pytest
+from skimage.morphology import label
+
+from topostats.tracing.nodestats import nodeStats
+
+# pylint: disable=redefined-outer-name
+
+
+@pytest.fixture()
+def network_array_representation_figure_8():
```

Comment:
Very minor but could add...

```suggestion
def network_array_representation_figure_8() -> npt.NDArray:
```

..assuming that `import numpy.typing as npt` is added to the imports.